optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/scope/PartitionChangeMove.java

```diff
@@ -134,8 +138,12 @@ public boolean isMoveDoable(ScoreDirector<Solution_> scoreDirector) {
             }
             destinationChangeMap.put(variableDescriptor, destinationPairList);
         }
-        return new PartitionChangeMove<>(destinationChangeMap);
+        return new PartitionChangeMove<>(destinationChangeMap, partIndex);
+    }
+
+    @Override
+    public String toString() {
+        return "partIndex=" + partIndex;
```

Comment:
(Not consistent with the rest of the toString() methods (they never tend to use "="), so I am adjusting this post-merge.)